workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfigurationEditorExtensions.cs

```diff
@@ -1,16 +1,14 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using System.Linq;
 using UnityEditor;
 using UnityEngine;
-using Object = UnityEngine.Object;
 
 namespace Improbable.Gdk.Tools
 {
     /// <summary>
     ///     Defines a custom inspector window that allows you to configure the GDK Tools.
     /// </summary>
-    [CustomEditor(typeof(ScriptableGdkToolsConfiguration))]
-    public class GdkToolsConfigurationInspector : Editor
+    public class GdkToolsConfigurationWindow : EditorWindow
```

Comment:
Potential for improvement: wondering if we should do a prompt to save file on disable of this window, or save at any modification? I like the latter more, fewer steps. The user can get unexpected results if they change the fields here but forget to hit the save button. The remaining problem is lack of undo - but that's something harder to implement for now.

Reply:
Probably don't want to save at any modification - as you can get into a bad state unintentionally. I prefer the prompt myself. 

We can look at this in a future PR as I'd like to get this in sooner rather than later so tests run in CI again 😬

Reply:
agreed, for now we should get this in asap to get tests running again